2022-12-15 03:32:09.951 +02:00 [ERR] Error: {"ClassName":"System.Exception","Message":null,"Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at SpendLess.UnitTests.Tests.<>c.<<TaskMethodsWithExceptionsAreLoggedByInterceptor>b__31_0>d.MoveNext() in D:\\C#\\zapas\\master\\SpendLess\\UnitTests\\Tests.cs:line 250\r\n--- End of stack trace from previous location ---\r\n   at SpendLess.Server.Interceptor.UnhandledExceptionLogger.InterceptAsync(Task task) in D:\\C#\\zapas\\master\\SpendLess\\Server\\Interceptor\\UnhandledExceptionLogger.cs:line 49","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":null,"HResult":-2146233088,"Source":"Tests","WatsonBuckets":null}
